nautobot-app/{{ cookiecutter.project_slug }}/docs/dev/dev_environment.md

```diff
@@ -74,7 +74,7 @@ nautobot-server migrate
 !!! note
     If you want to develop on the latest develop branch of Nautobot, run the following command: `poetry add --optional git+https://github.com/nautobot/nautobot@develop`. After the `@` symbol must match either a branch or a tag.
 
-You can now run `nautobot-server` commands as you would from the [Nautobot documentation](https://nautobot.readthedocs.io/en/latest/) for example to start the development server:
+You can now run `nautobot-server` commands as you would from the [Nautobot documentation](https://docs.nautobot.com/en/stable/) for example to start the development server:
```

Comment:
Should this link perhaps be https://docs.nautobot.com/projects/core/en/stable/user-guide/administration/tools/nautobot-server/ ?